engine/apps/api/serializers/alert_receive_channel_connection.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import serializers
+
+from apps.alerts.models import AlertReceiveChannel, AlertReceiveChannelConnection
+from apps.api.serializers.alert_receive_channel import FastAlertReceiveChannelSerializer
+
+
+class AlertReceiveChannelSourceChannelSerializer(serializers.ModelSerializer):
+    alert_receive_channel = FastAlertReceiveChannelSerializer(source="source_alert_receive_channel", read_only=True)
+    backsync = serializers.BooleanField()
+
+    class Meta:
+        model = AlertReceiveChannelConnection
+        fields = ["alert_receive_channel", "backsync"]
```

Comment:
Should `backsync` field be only on the connected side? (it only makes sense for a (source + connected) combination, doesn't it?)

Reply:
hmm, I think `backsync` makes sense for both the "connected" and "source" fields. the first one is "what integrations are connected to this integration?" and the second is "what integrations this integration is connected to?". so `backsync` applies to both, right?

Reply:
here's an example: https://github.com/grafana/oncall/blob/48038438c88b173e3fbabad704d02e485a72d73e/engine/apps/api/tests/test_alert_receive_channel.py#L1856-L1911

the backsync value is the same for both, the only thing that changes is the direction of this relationship. does this make sense?